docs/csharp/how-to/compare-strings.md

```diff
@@ -1,87 +1,69 @@
 ---
 title: "How to compare strings - C# Guide"
-description: Learn how to compare and order string values, with or without case, with or without culture specific ordering
-ms.date: 02/15/2022
+description: Learn how to compare and order string values, with or without case, with or without culture specific ordering.
+ms.date: 03/15/2024
 helpviewer_keywords:
     - "strings [C#], comparison"
     - "comparing strings [C#]"
 ---
 
 # How to compare strings in C\#
 
-You compare strings to answer one of two questions: "Are these two strings
-equal?" or "In what order should these strings be placed when sorting them?"
+You compare strings to answer one of two questions: "Are these two strings equal?" or "In what order should these strings be placed when sorting them?"
 
-Those two questions are complicated by factors that affect string comparisons:
+Multiple factors that affect string comparisons complicate those two questions:
```

Comment:
I like the original version better, although I know Acrolinx probably doesn't.

Reply:
I did too:

```suggestion
Those two questions are complicated by factors that affect string comparisons:
```